did_webvh/core/problem_details.py

```diff
@@ -0,0 +1,90 @@
+"""Support for IETF problem details."""
+
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass
+class ProblemDetails:
+    """RFC 9457 problem details instance."""
+
+    type: str
+    title: str | None = None
+    detail: str | None = None
+    status: int | None = None
+    exts: dict[str, Any] | None = None
+
+    DEFAULT_SPEC_URI = "https://didwebvh.info/main/resolution-errors/"
+
+    @classmethod
+    def invalid_did(cls, detail: str = None, **exts):
+        """Instantiate a generic invalid DID error."""
+        return cls(
+            type="https://w3id.org/security#INVALID_CONTROLLED_IDENTIFIER_DOCUMENT_ID",
```

Comment:
Presumably the type server should be a variable so it is easy to adjust.  I should have the document in the right place soon https://didwebvh/security

Reply:
Or is this intentionally pointing to the w3id server?  I gather that is what you intended...

Reply:
That one is referenced on the main page of the implementation guide, maybe it's supposed to be something else. There's still some alignment to do on the error codes, I use a couple that aren't in the guide.

Reply:
I got rid of that error message and added another one for issues with the DID document itself.